javascript/javascript3/week3/index.js

```diff
@@ -2,37 +2,38 @@
 
 // Class creation time!
 class Circle {
-  constructor(x, y, r, startAngle, endAngle, fillColor) {
-    this.x = x;
-    this.y = y;
-    this.r = r;
-    this.startAngle = startAngle;
-    this.endAngle = endAngle;
-    this.fillColor = fillColor;
-  }
-  draw() {
-    const canvas = document.getElementById("circle");
-    canvas.style.width = "100%";
-    canvas.style.height = "100%";
-    const ctx = canvas.getContext("2d");
-    ctx.beginPath();
-    ctx.arc(this.x, this.y, this.r, this.startAngle, this.endAngle);
-    ctx.fillStyle = this.fillColor;
-    ctx.stroke();
-    ctx.fill();
-  }
+    constructor(x, y, r, startAngle, endAngle, fillColor) {
+        this.x = x;
+        this.y = y;
+        this.r = r;
+        this.startAngle = startAngle;
+        this.endAngle = endAngle;
+        this.fillColor = fillColor;
+    }
+
+    draw(ctx) {
+        canvas.style.width = "100%";
+        canvas.style.height = "100%";
```

Comment:
Excellent!!
It is better to set  "width & height"  where you get the "canvas" with "getElementById".

const canvas = document.getElementById("circle");
canvas.style.width = "100%";
canvas.style.height = "100%";